base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbt.java

```diff
@@ -203,45 +204,53 @@ public Image getNextTurnIcon() {
     }
 
     /**
-     * Sets a Fraction of distance till next maneuver
+     * Sets the distanceToManeuver.
      *
-     * @param distanceToManeuver a Double value representing a Fraction of distance till next maneuver
-     *                           <p></p>
-     *                           <b>Notes: </b>Minvalue=0; Maxvalue=1000000000
+     * @param distanceToManeuver Distance (in meters) until next maneuver. May be used to calculate progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public ShowConstantTbt setDistanceToManeuver(Double distanceToManeuver) {
-        setParameters(KEY_MANEUVER_DISTANCE, distanceToManeuver);
+        setParameters(KEY_DISTANCE_TO_MANEUVER, distanceToManeuver);
         return this;
     }
 
     /**
-     * Gets a Fraction of distance till next maneuver
+     * Gets the distanceToManeuver.
      *
-     * @return Double -a Double value representing a Fraction of distance till next maneuver
+     * @return Float Distance (in meters) until next maneuver. May be used to calculate progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public Double getDistanceToManeuver() {
-        return getDouble(KEY_MANEUVER_DISTANCE);
+        Object object = getParameters(KEY_DISTANCE_TO_MANEUVER);
```

Comment:
@kostyaBoss I would delete lines 226 and 227 and replace it with the old code `return getDouble(KEY_MANEUVER_DISTANCE);`.